components/Navigation/index.js

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link";
+import styled from "styled-components";
+import ProductSvg from "@/public/images/products_icon.svg";
+import StoreSvg from "@/public/images/stores_icon.svg";
+
+import { useRouter } from "next/router";
+
+const StyledNavBar = styled.ul`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin: 0;
+`;
+
+const StyledNavLink = styled(Link)`
+  text-decoration: none;
+  display: inline-block;
+  padding: 10px;
+  width: 100%;
+  border-top: ${({ $currentPage }) =>
+    $currentPage
+      ? "solid 3px var(--accentColor)"
+      : "solid 3px var(--secondaryBackgroundColor)"};
+  &:hover {
+    border-color: ${({ $currentPage }) => !$currentPage && "gray"};
+  }
+`;
+
+const StyledListItem = styled.li`
+  width: 50%;
+  text-align: center;
+`;
+
+const StyledProductSvg = styled(ProductSvg)`
+  path {
+    fill: ${({ $currentPage }) =>
+      $currentPage ? "var(--accentColor)" : "var(--primaryDarkColor)"};
+  }
+`;
+
+const StyledStoreSvg = styled(StoreSvg)`
+  path {
+    fill: ${({ $currentPage }) =>
+      $currentPage ? "var(--accentColor)" : "var(--primaryDarkColor)"};
+  }
+`;
+
+export default function Navigation() {
+  const router = useRouter();
+
+  return (
+    <nav>
+      <StyledNavBar>
+        <StyledListItem>
+          <StyledNavLink href="/" $currentPage={router.pathname === "/"}>
+            <StyledProductSvg $currentPage={router.pathname === "/"} />
```

Comment:
Why do you use a ``$`` in your prop name?

Reply:
We use the $ Sign because $currentPage ist only use for Styling:

From the Handout:

> You can adapt styling based on props. To do so, you need to pass the props to the styled component. Most of the time you'll want to prefix the prop with a $. This tells styled components, that the prop should not be passed to the underlying DOM element or component and should only be used for styling.

Reply:
Thank you for explaining. I lost this little detail anywhere in my mind. Great that you got it 💪